frontend/components/navbar/NavbarLinks.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div
+    :class="cn(navbarLinksVariants({ variant }), props.class)"
+  >
+    <NavbarLink url="#" :variant="firstLinkVariant">
+      Landsat Locator
+    </NavbarLink>
+    <NavbarLink url="#" :variant="linksVariant">
+      Search for data
+    </NavbarLink>
+    <NavbarLink url="#" :variant="linksVariant">
+      About us
+    </NavbarLink>
+    <img
+      v-if="isUserLoggedIn" h="5vh" alt="User profile picture" rounded-full
+      :src="isUserLoggedIn ? user?.user_metadata.avatar.url : ''"
+      :class="isUserLoggedIn ? user?.user_metadata.avatar.url ? '' : 'i-carbon:user-avatar-filled' : ''"
+    >
+    <NavbarLink v-if="!isUserLoggedIn" url="/register" :variant="linksVariant">
+      Sign in
+    </NavbarLink>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { HTMLAttributes } from "vue";
+import { type NavbarLinksVariants, navbarLinksVariants } from ".";
+import { cn } from "@/lib/utils";
+
+interface Props {
+  variant?: NavbarLinksVariants["variant"]
+  linksVariant?: NavbarLinksVariants["linksVariant"]
+  firstLinkVariant?: NavbarLinksVariants["firstLinkVariant"]
+  class?: HTMLAttributes["class"]
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  as: "navbar-links"
+});
+
+const isUserLoggedIn = ref(false);
+
+const user = useSupabaseUser();
+
+if (user.value !== null) {
```

Comment:
I think, you can just just `v-if="user"` instead of manually setting variable isLoggedIn